try {
    throw new Error('Algo sali√≥ mal');
} catch (error) {
    console.log(error);
}